packages/terra-overlay/tests/jest/LoadingOverlay.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { IntlProvider } from 'react-intl';
+import LoadingOverlay from '../../src/LoadingOverlay';
+import messages from '../../translations/en-US.json';
+
+const locale = 'en-US';
+
+describe('LoadingOverlay', () => {
+  it('should render a null component when isOpen is not provided', () => {
+    const wrapper = shallow(<IntlProvider locale={locale} messages={messages}><LoadingOverlay /></IntlProvider>);
```

Comment:
nit: I'd recommend using `render` over `shallow` so we have more context in the snapshot.